data.sql

```diff
@@ -5,3 +5,10 @@ INSERT INTO animals VALUES (1,'Agumon', '02-03-2020',0,true,10.23);
 INSERT INTO animals VALUES (2,'Gabumon', '11-15-2018',2,true,8);    
 INSERT INTO animals VALUES (3,'Pikachu', '01-07-2021',1,false,15.04);
 INSERT INTO animals VALUES (4,'Devimon', '05-12-2017',5,true,11);
+INSERT INTO animals VALUES (5,'Charmander', '02-08-2020',0,false,11);
+INSERT INTO animals VALUES (6,'Plantmon', '11-15-2021',2,true,5.7);
+INSERT INTO animals VALUES (7,'Squirtle', '04-02-1993',3,false,12.13);
+INSERT INTO animals VALUES (8,'Angemon', '06-12-2015',1,true,45);
+INSERT INTO animals VALUES (9,'Boarmon', '06-07-2005',7,true,20.4);
+INSERT INTO animals VALUES (10,'Blossom', '10-13-1998',3,true,17);
+INSERT INTO animals VALUES (11,'Ditto', '05-14-2022',4,true,22);
```

Comment:
- [ ] Kindly When storing a date value, PostgreSQL uses the `yyyy-mm-dd format` e.g. 1994-10-27. This format is also used in PostgreSQL for inserting data. 
For more documentation visit this l[ink ](https://blog.devart.com/postgres-date-types-and-functions-formatting.html)


- [ ] Kindly let's be more careful. According to the Project requirements, you should Insert the following data 👇🏻  As you can see the fourth first data has a negative `weight_kg` value, kindly let's respect this 💯 👍🏻  
- Animal: His name is Charmander. He was born on Feb 8th, 2020, and currently weighs -11kg. He is not neutered and he has never tried to escape.
- Animal: Her name is Plantmon. She was born on Nov 15th, 2021, and currently weighs -5.7kg. She is neutered and she has tried to escape 2 times.
- Animal: His name is Squirtle. He was born on Apr 2nd, 1993, and currently weighs -12.13kg. He was not neutered and he has tried to escape 3 times.
- Animal: His name is Angemon. He was born on Jun 12th, 2005, and currently weighs -45kg. He is neutered and he has tried to

```sql
   INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Charmander', '2020/2/8', '0', 'false', '-11.0');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Plantmon', '2021/11/15', '2', 'true', '-5.7');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Squirtle', '1993/04/02', '3', 'false', '-12.13');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Angemon', '2005/06/12', '1', 'true', '-45.0');
```

Reply:
Thank you for the kind review, I have updated the data as requested and saved the queries into the queries.sql.
Following snapshot shows the data available inside the animals table now:

![chanages](https://user-images.githubusercontent.com/43011742/199578916-7f3ae0d3-9df7-491b-8aba-1e893941795f.PNG)